doc/enterprise/overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+![alt tag](enterprise.png)
+
+Pachyderm Enterprise Edition includes everything you need to scale and manage Pachyderm data pipelines in an enterprise setting. It is secure and simple, and it delivers the most recent version of Pachyderm along with:
+
+- Administrative and security features needed for enterprise-scale implementations of Pachyderm 
+- Visual and interactive interfaces to Pachyderm  
+
+Pachdyerm Enterprise Edition can be deployed easily on top of an existing or new deployment of Pachyderm, and we have engineers available to help enterprise customers get up and running very quickly.  To get more information about Pachyderm Enterprise Edition, to ask questions, or to get access for evaluation, please contact us at [sales@pachyderm.io](mailto:sales@pachyderm.io) or on our [Slack](http://slack.pachyderm.io/). 
+
+## Pipeline Visualization and Data Exploration
+
+![alt tag](dashboard.png)
+
+Pachyderm Enterprise Edition includes a full UI, or dashboard, for visualizing your pipelines and exploring your data.  Pachyderm Enterprise will automatically infer the structure of your Pachyderm DAGs and display them to you visually.  You can even click on individual segments of the pipelines to see what data is being processed, how many jobs have run, what images and commands are being run in your pipeline stages, and much more!  You can also explore the versioned data in your Pachyderm data respositories and see the commit history of your data.  
+
+## Administrative Controls, Interactive Pipeline Configuration*
+
+With Pachyderm Enterprise, you don't have to rely solely on command line tools and language libraries to configure and control Pachyderm. With new versions of our UI (currently in beta) you can control, scale, and configure Pachyderm interactively.
+
+## Access Controls*
+
+Enterprise-scale deployments require access controls and multitenancy.  Pachyderm Enterprise Edition gives you the ability to control access to your production pipelines, data, and configuration, so you can silo data, prevent unintended modficiations to production pipelines, and support multiple data scientists or even mutliple data science groups.
+
+## Advanced Statistics* 
+
+Pachyderm Enterprise Edition gives you advanced insights into your data, jobs, and results.  For example, you can see how much time your jobs spend downloading/uploading data, what data was mounting in workers for processing and what that data looked like when it was mounted, portions of data that were not processed, and much more.  You can explore this information programmatically or via a number of charts and plots that help you parse the information quickly.
```

Comment:
"For example, you can see how much time your jobs spend downloading/uploading data, what data was mounting in workers for processing and what that data looked like when it was mounted, portions of data that were not processed, and much more" --> List too long. 

perhaps --> "For example, you can see how much time your jobs spend downloading/uploading data, what data was processed or skipped, and which workers were given particular datums."